src/qt/bitcoin.cpp

```diff
@@ -508,9 +506,9 @@ int GuiMain(int argc, char* argv[])
     /// 5. Now that settings and translations are available, ask user for data directory
     // User language is set up: pick a data directory
     bool did_show_intro = false;
-    bool prune = false; // Intro dialog prune check box
+    int64_t prune_MiB = 0;  // Intro dialog prune configuration
```

Comment:
style piconit: [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) suggests
```suggestion
    int64_t prune_MiB = 0; // Intro dialog prune configuration
```